scraper/visualize.py

```diff
@@ -157,16 +158,18 @@ def get_products_from_master_products(master_products: Iterable[MasterProduct])
     return [product for master_product in master_products for product in master_product.products]
 
 
-def config_figure(figure: go.Figure, figure_title: str) -> None:
-    figure.update_traces(mode="markers+lines")
+def config_figure(figure: go.Figure, figure_title: str) -> None:   
     figure.update_layout(
         title=figure_title,
         xaxis_title="Date",
         yaxis_title="Price",
-        hovermode="x",
-        separators=".,",
+        hovermode="closest",
+        separators=".,",   
+        legend=dict(orientation = "h", yanchor="bottom",y=-2,xanchor="left", x=0),
```

Comment:
This seems make the plot have a fixed height even when few products are shown.
![billede](https://github.com/Crinibus/scraper/assets/57172157/ae494602-bcb4-4123-8b06-419cd5b23f51)

Removing `yanchor="bottom, y=-2, x=0"` fixes this, but then when many products are shown the legends overlap the x-axis title and I can't seem to find a solution to this.
![billede](https://github.com/Crinibus/scraper/assets/57172157/0b598131-29ac-4a7e-85a5-9ac1275452f7)